runner/lib/main.js

```diff
@@ -162,10 +164,19 @@ const makeInterrupterKit = ({ console }) => {
  * @returns {Promise<import('./tasks/types.js').SDKBinaries>}
  */
 const getSDKBinaries = async () => {
-  const srcHelpers = 'agoric/src/helpers.js';
+  const helpersSource = 'src/helpers.js';
+  const srcHelpers = `agoric/${helpersSource}`;
   const libHelpers = 'agoric/lib/helpers.js';
   try {
-    const cliHelpers = await import(srcHelpers).catch(() => import(libHelpers));
+    const cliHelpers = await import(srcHelpers)
+      .catch(() => import(libHelpers))
+      .catch((e) =>
+        process.env.SDK_SRC
+          ? import(
+              `${process.env.SDK_SRC}/packages/agoric-cli/${helpersSource}`
+            )
+          : Promise.reject(e),
+      );
```

Comment:
because `SDK_SRC` is option someone has to enable, I would expect it to take precedence.

also I would suggest refactoring to,
```suggestion
    const cliHelpers = await getCliHelpers();
```

It's a shame we have to deep import instead of having real exports of that package, but even if we added good exports we'd still need backwards compatibility so it would only add another case to this code.

Reply:
In the past I've thought of adding a command to the agoric-cli to return these path. However I don't think it would help in this case since `agoric` would not be on the PATH. I'm open to suggestion on how to find entrypoint paths in agoric-sdk

Reply:
> because `SDK_SRC` is option someone has to enable, I would expect it to take precedence.

Yeah you're right. I originally implemented it as a fallback, but it should take precedence.